docs/docs/api/turn-by-turn/api-reference.md

```diff
@@ -367,7 +367,8 @@ For example a bus request with the result in Spanish using the OSRM (Open Source
 | `prioritize_bidirectional` | Prioritize `bidirectional a*` when `date_time.type = depart_at/current`. By default `time_dependent_forward a*` is used in these cases, but `bidirectional a*` is much faster. Currently it does not update the time (and speeds) when searching for the route path, but the ETA on that route is recalculated based on the time-dependent speeds |
 | `roundabout_exits` | A boolean indicating whether exit instructions at roundabouts should be added to the output or not. Default is true. |
 | `admin_crossings` | When present and `true`, the successful route summary will include the two keys `admins` and `admin_crossings`. `admins` is an array of administrative regions the route lies within. `admin_crossings` is an array of objects that contain `from_admin_index` and `to_admin_index`, which are indices into the `admins` array. They also contain `from_shape_index` and `to_shape_index`, which are start and end indices of the edge along which an administrative boundary is crossed. |
-| `turn_lanes` | When present and `true`, each maneuver in the route response can include a `lanes` array describing lane-level guidance. The lanes array details possible `directions`, as well as which lanes are `valid` or `active` for following the maneuver.
+| `turn_lanes` | When present and `true`, each maneuver in the route response can include a `lanes` array describing lane-level guidance. The lanes array details possible `directions`, as well as which lanes are `valid` or `active` for following the maneuver. |
+| `reverse_time_tracking` | Which time tracking strategy to use on the non-time aware expansion in the bidirectional routing algorithm: `invalid` will assume no time, `heuristic` will try a best guess based on the bee line distance between origin and destination. Default `heuristic` |
```

Comment:
As long as we don't have a battle tested heuristic, users should be able to opt out of it and choose the more conservative method.

Reply:
should we mark it BETA? IMO ideally we'd remove this at some point when we either have a good feeling with a simpler heuristic or, less likely, make a more sophisticated one.